### ISSUE TITLE
[Bug]: Removing Discord Rich Presence breaks client

### ISSUE THREAD

**Issue by phit** [FTBTeam/FTB-Modpack-Issues#1540]
### Modpack

Stoneblock 3

### Modpack version

1.0.0

### Log Files

snippet
```
[01Nov2022 21:19:41.118] [Render thread/ERROR] [net.minecraftforge.eventbus.EventBus/EVENTBUS]: Exception caught during firing event: ReferenceError: "SDRP" is not defined. (startup_scripts:sdrp.js#17)
	Index: 23
	Listeners:
		0: HIGHEST
		1: ASM: class dan200.computercraft.shared.util.DropConsumer onEntitySpawn(Lnet/minecraftforge/event/entity/EntityJoinWorldEvent;)V
		2: ASM: mekanism.common.CommonWorldTickHandler@70
```

### Describe the bug

your player is unable to move and you are invisible

### Steps to reproduce

* remove Discord Rich Presence from the mods folder because you don't like sending Discord more data than necessary
* kubejs/startup_scripts/sdrp.js crashes

### Expected behaviour

script should fail gracefully, and not break your client

### Screenshots

player stuck and invisible
![image](https://user-images.githubusercontent.com/2097483/199337267-f632d0a5-1a42-4156-ae07-dd71c9fa3dec.png)

**Comment by Sunekaer** [comment]
Yes, because there is a script there interacts, it is not possible to fail gracefully, as it attaches to a forge event.
fyi We dont support modification of our packs :)

**Comment by phit** [comment]
that's unfortunate, is there no way to check if the class is defined in KubeJS? for myself I can just delete the file, but I suspect I'm not the only person that always removes that mod

**Comment by phit** [comment]
thank you for fixing it anyway!
